spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-graph-terms">
+    <h3>Graph Terms</h3>
 
-    <p>A <a>quoted triple</a> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>graph term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF graph</a>, which can be used as the <a>subject</a> or <a>object</a>
       of another triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>A <a>graph term</a> is not necessarily asserted, allowing
```

Comment:
It is not clear to me what it would mean for a graph term to be asserted or not. I think, here it is better to talk about the triples in a graph term. Hence, the following change suggestion.
```suggestion
    <p>The triples in a <a>graph term</a> are not necessarily asserted, allowing
```